build.rs

```diff
@@ -1,4 +1,41 @@
+use cargo_toml::{Dependency, Manifest};
+use std::env;
+use std::path::Path;
+
 pub fn main() {
+    let manifest_path =
+        Path::new(env!("CARGO_MANIFEST_DIR")).join("Cargo.toml");
+    let manifest = Manifest::from_path(manifest_path).expect("Can't panic");
+
+    let set_dep_env = |key: &str| {
```

Comment:
Inspired by RET, used to get `build_information` endpoint, so that Android/iOS can get info about dependencies inside of Sargon.